QMLComponents/controls/jaspcontrol.cpp

```diff
@@ -793,6 +793,9 @@ void JASPControl::_notifyFormOfActiveFocus()
 	if (!hasActiveFocus())
 		_hasActiveFocus = false;
 
-	if (_form)
+	//All JASP controls are focusscopes when they receive focus due to a child being clicked the reason is 8 for some reason (undocumented as of 3-1-2023)
+	//Objects like jasp groupboxes could unrightfully be marked as the active jasp control when a child jasp control gets focus
+	//For this reason we check if the focus reason is the result of user input or the focus scope system.
+	if (_form && _focusReason >= Qt::MouseFocusReason && _focusReason <= Qt::OtherFocusReason)
```

Comment:
Sorry, I don't understand this extra check : the only focus reason that cannot be between MouseFocusReason and OtherFocusReason is NoFocusReason (also, as this function is called only when the focus changes, that would be really strange).

Reply:
There are hidden values for the FocusReason Enum that do not concern user input.
Such as when a FocusScope gets focus due to a focus event of  one of its children (value 8) .
You can check it under the debugger. It is weird but true.